### PR TITLE
Test

### PR DESCRIPTION
**Pull request by vaishakp**
Setup test environment + init with SXS modes test

### CODE REVIEW

**Comment by prayush** [comment]
Looks good @vaishakp. Do you want to squash and rebase?

File: docs/environment.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: nrcat_test
```
Comment (prayush) [inline docs/environment.yml:1]: perhaps rename this to 'nrcatalogs' or something similar, without 'test'.

File: test/testSXSModes.py [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+import matplotlib.pyplot as plt
+from nrcatalogtools.sxs import SXSCatalog
+# from pycbc.waveform.utils import coalign_waveforms
+from pycbc import pnutils
+from pycbc.filter.matchedfilter import match
+# pycbc
+# from pycbc.waveform import td_approximants
+from pycbc.types.timeseries import TimeSeries
+# waveformtools
+from waveformtools.waveforms import modes_array
```
Comment (prayush) [inline test/testSXSModes.py:39]: `waveformtools` should be added as a dependency to `requirements.txt` and `environment.yaml`

File: test/testSXSModes.py [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+from waveformtools.waveformtools import (interp_resam_wfs, message,
+                                         xtract_camp_phase)
+
+# unittest helper funcs
+from helper import *
+
+home = str(Path.home())
```
Comment (prayush) [inline test/testSXSModes.py:46]: repeated from line 20 above

File: test/testSXSModes.py [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+    # plt.grid()
+    # plt.show()
+
+    # fig, ax = plt.subplots()
+    # ax.set_yscale('log')
+    # ax.plot(taxis, np.absolute(dphase))
+    # ax.set_title(f'Phase diff l{ell}m{emm}')
+    # plt.grid()
+    # plt.show()
+
```
Comment (prayush) [inline test/testSXSModes.py:171]: If all these commented out lines are not needed, remove them pls. But if they are, perhaps let them be behind a boolean flag that can be switched?

File: test/testSXSModes.py (hunk 2 above)
Comment (vaishakp) [reply test/testSXSModes.py:39]: I haven't published my repo to pypi. Do you suggest I do it?

Comment (prayush) [reply test/testSXSModes.py:39]: If its public already, then why not?

**Pull request merged**